src/Aspire.Hosting/Dcp/DcpDependencyCheck.cs

```diff
@@ -106,6 +106,10 @@ public DcpDependencyCheck(IOptions<DcpOptions> dcpOptions)
                 _dcpInfo = dcpInfo;
                 return dcpInfo;
             }
+            catch (OperationCanceledException) when (cancellationToken.IsCancellationRequested)
```

Comment:
What is the purpose of catching and then doing nothing but re-throwing this exception?